shell/platform/embedder/embedder_external_view_embedder.cc

```diff
@@ -196,6 +202,19 @@ bool EmbedderExternalViewEmbedder::SubmitFrame(GrContext* context) {
       return false;
     }
 
+    // Indicate a layer for the platform view. Add to `presented_platform_views`
+    // in order to keep at allocated just for the scope of the current method.
+    // The layers presented to the embedder will contain a back pointer to this
+    // struct. It is safe to deallocate when the embedder callback is done.
+    presented_platform_views[view_id] = MakePlatformView(view_id);
+    presented_layers.push_back(
+        MakeLayer(params, presented_platform_views.at(view_id)));
+
+    if (!pending_canvas_spies_[view_id]->DidDrawIntoCanvas()) {
```

Comment:
Use `.at()` instead of `.[]` when you know you don't want to insert a default constructed map element. We know this is safe because of the DCHECK added above.